meshnet/server.go

```diff
@@ -2704,3 +2743,21 @@ func (s *Server) GetPrivateKey(ctx context.Context, _ *pb.Empty) (*pb.PrivateKey
 		},
 	}, nil
 }
+
+// refreshIncoming will block and unblock firewall rules for incoming traffic, applying routing, icoming and local network
+// access rules
+func (s *Server) refreshIncoming(peer mesh.MachinePeer) error {
```

Comment:
This does not need any information from the server and it is always used in combination with `netw.ResetRouting`. Would it make sense to move this to networker and merge those methods?